monochart/templates/_helpers.tpl

```diff
@@ -122,6 +122,13 @@ VolumeMounts template block for deployable resources
 {{- if not ( empty $config.files ) }}
 - mountPath: {{ default (printf "/%s" $name) $config.mountPath }}
   name: config-{{ $name }}-files
+{{/*
+If the mount path ends with one of extensions specified below - we assume it's a regular file so
+we will mount it as a regular file (not a directory).
+  */}}
+{{- if regexMatch ".*.(json|yaml|txt)$" $config.mountPath }}
+  subPath: {{ regexFind "[a-zA-Z0-9_.-]*.(json|yaml|txt)$" $config.mountPath }}
+{{- end }}
```

Comment:
I'm not 100% sure this is needed; each file in a configmap should be mounted in the dir as a file the same name as the key in the configmap, to my understanding. Are you sure this part is required? Looking above on lines 123-124, I think this should be enough to make it work.

Reply:
the "subPath" setting is necessary for mounting a file. When I was trying to mount a config.json file only with `mountPath: /www/config.json` the file was available under `/www/config.json/config.json`